packages/shared/src/variants/s_fractional.ts

```diff
@@ -327,11 +358,13 @@ export function sFractionalMoveColors(
   return [primaryColor, secondaryColor];
 }
 
-export const sFractionalVariant: Variant<SFractionalConfig> = {
-  gameClass: SFractional,
-  defaultConfig: SFractional.defaultConfig,
-  description:
-    "Stones have a secondary colour that also requires liberties to avoid capture.",
-  rulesDescription: sfractionalRulesDescription,
-  getPlayerColors: Baduk.getPlayerColors,
-};
+export const sFractionalVariant: Variant<SFractionalConfig, SFractionalState> =
+  {
+    gameClass: SFractional,
+    defaultConfig: SFractional.defaultConfig,
+    description:
+      "Stones have a secondary colour that also requires liberties to avoid capture.",
+    rulesDescription: sfractionalRulesDescription,
+    getPlayerColors: Baduk.getPlayerColors,
+    uiTransform: SFractional.uiTransform,
+  };
```

Comment:
Why is this whole part changed? Was there some problem with indentation or is there now?

Reply:
My auto formatter does this. It put the `{` bracked into the next line, because SFractionalState was added in the generics.

It also added indentation, I believe as an aftereffect of the above.

Reply:
I think it's just the formatter adding whitespace (notice the `Variant<...>` part has been expanded)

Reply:
Ah, maybe the line got too long, so Prettier put the curly bracket on the next line, then the indentation makes sense.